starlette/routing.py

```diff
@@ -440,6 +449,12 @@ def __eq__(self, other: typing.Any) -> bool:
             and self.app == other.app
         )
 
+    def __repr__(self) -> str:
+        return (
+            f"<{self.__class__.__name__}: "
+            f"path={self.path}, name={self.name or ''}, app={self.app}>"
```

Comment:
👍 
```suggestion
            f"path={self.path}, name={self.name or ''}, app={self.app!r}>"
```

Reply:
Actually, the `!r` is used on all parameters in the other `__repr__` that we have in code 🤔

Reply:
Ah, I got why... It's because with them, there's no need to wrap on parenthesis!